### DISCUSSION TITLE
CartesianChart EEG Behavior

### DISCUSSION THREAD

**Discussion by ManuelKetisch** [Live-Charts/LiveCharts2#1430] [Q&A]
Hello, i am new to LiveCharts2 and i would like to know, if anyone has ever used this library for medical data display like EEG data.
I am planning on using it with AvaloniaUI for such a use case. Does anyone have any experience in such a case? 
How would i go about configuring the CartesianChart to resemble an eeg data visualization?
Is this library even a good pick, or should i choose something else?

**Comment by Daimonion1980** [comment]
Are you able to provide a screenshot or a video what you mean with eeg data visualization?
